mod_cluster/extension/src/main/resources/schema/jboss-as-mod-cluster_4_0.xsd

```diff
@@ -301,17 +301,25 @@
                 </xs:annotation>
             </xs:element>
         </xs:choice>
+        <xs:attribute name="decay" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>
+                    The factor by which a historic load values should degrade in significance.
```

Comment:
"The factor by which historic load values should degrade in significance."

Reply:
Fixed.